ports/espressif/common-hal/busio/I2C.c

```diff
@@ -58,6 +58,9 @@ void common_hal_busio_i2c_construct(busio_i2c_obj_t *self,
     gpio_pulldown_dis(sda->number);
     gpio_pulldown_dis(scl->number);
 
+    // This appears to be a board-specific way of supporting internal pullups without 
+    // going through all of the changes I've made - @dhalbert can you comment?
+    // Not going to change anything here before discussion - will just throw internal_pullup away
```

Comment:
Make a similar change as suggested in the nrf port:
```c
if (internal_pullup) {
    // set the pullups
} else {
    // test for pullups
}
```

Reply:
Ok, so do away with the existing code beneath this comment?

```
// This appears to be a board-specific way of supporting internal pullups without 
    // going through all of the changes I've made - @dhalbert can you comment?
    // Not going to change anything here before discussion - will just throw internal_pullup away
    #if CIRCUITPY_I2C_ALLOW_INTERNAL_PULL_UP
    gpio_pullup_en(sda->number);
    gpio_pullup_en(scl->number);
    #endif
```

Reply:
This is in `ports/espressif/common-hal/busio/I2C.c` around line 64.

Reply:
Ok, so remove the `#if` and wrap the contents into the `if-else`. Can do.